src/components/common/input/CustomDateInput.tsx

```
@@ -0,0 +1,39 @@
+import { faCalendar } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { forwardRef } from 'react';
+import cn from '@/utils/cn';
+
+type Props = {
+  value?: string;
+  onClick?: () => void;
+  isFocus: boolean;
+  setIsFocus: (focused: boolean) => void;
```

ESLint

[error] 'focused' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)